frontend/src/components/charts/doughnut.js

```diff
@@ -41,9 +47,10 @@ class Doughnut extends Component {
                     </div>
                     <div className="footer">
                         <div className="legend">
-                            <div><i className="fa fa-circle text-info" /> { this.props.data.labels[0] } </div>
-                            <div><i className="fa fa-circle text-warning" /> { this.props.data.labels[1] } </div>
-                            <div><i className="fa fa-circle text-danger" /> { this.props.data.labels[2] } </div>
+                            <div><i className="fa fa-circle text-info" /> { this.props.data.labels[0] } <Badge>{this.props.data.datasets[0].data[0]}</Badge></div>
```

Comment:
for me the chart make no sense at all. if anything it should be a bar-chart with their legend+value below.. requireing me to use the mouse to get basic information is imo not good UX :)